tests/smoketest/smoketest.sh

```diff
@@ -82,21 +53,19 @@ done
 echo
 
 echo "*** [INFO] Logs from qdr..."
-oc logs "$(oc get pod -l application=qdr-white -o jsonpath='{.items[0].metadata.name}')"
+oc logs "$(oc get pod -l application=saf-default-interconnect -o jsonpath='{.items[0].metadata.name}')"
 echo
 
 echo "*** [INFO] Logs from smart gateways..."
-for NAME in "${CLOUDNAMES[@]}"; do
-    oc logs "$(oc get pod -l "deploymentconfig=${NAME}-smartgateway" -o jsonpath='{.items[0].metadata.name}')"
-done
+oc logs "$(oc get pod -l "deploymentconfig=saf-default-telemetry-smartgateway" -o jsonpath='{.items[0].metadata.name}')"
 echo
 
 echo "*** [INFO] Logs from smart gateway operator..."
-oc logs "$(oc get pod -l app=smart-gateway-operator -o jsonpath='{.items[0].metadata.name}')"
+oc logs "$(oc get pod -l app=smart-gateway -o jsonpath='{.items[0].metadata.name}')" -c operator
```

Comment:
I'm fine with this as it will return as a stream, but the `-c ansible` container will produce prettier output. Just FYI in case a nicer output makes sense here.

Reply:
Alright, with switch to `ansible`